plugins/CoreHome/Tracker/VisitRequestProcessor.php

```diff
@@ -189,7 +190,8 @@ public function isVisitNew(VisitProperties $visitProperties, Request $request, $
             return true;
         }
 
-        if (!$this->lastUserIdWasSetAndDoesMatch($visitProperties, $request)) {
+        if (!TrackerConfig::getConfigValue('enable_userid_overwrites_visitorid')
```

Comment:
I suppose this could also be a separate setting? Or not sure if that makes sense or not

Reply:
For now I would keep it simple and leave it like that.